public/src/main/java/com/revenuecat/purchases/models/GoogleSubscriptionOption.kt

```diff
@@ -1,11 +1,27 @@
 package com.revenuecat.purchases.models
 
+import com.android.billingclient.api.ProductDetails
+
 /**
  * Defines an option for purchasing a Google subscription
  */
 data class GoogleSubscriptionOption(
-    override val id: String,
+    val productId: String,
+    val basePlanId: String,
+    val offerId: String?,
     override val pricingPhases: List<PricingPhase>,
     override val tags: List<String>,
+    val productDetails: ProductDetails,
+    val offerToken: String
+) : SubscriptionOption {
+    override val id: String
+        get() = basePlanId + if (offerId.isNullOrBlank()) "" else ":$offerId"
```

Comment:
Hmm it's interesting the id doesn't include the productId. There could probably be more than one subscription option with the same id but different product id right? But I don't think this is a change, so it's probably ok?

Reply:
I agree that it feels like it would be nice if it was a full scoped id but this is how it was so don't really want to break it 😛